src/backend/base/langflow/components/tools/file_manipulation.py

```
@@ -0,0 +1,1322 @@
+                for match in pattern.finditer(content):
+                    start, end = match.span()
+                    line_count = content[:start].count("\n") + 1
+                    occurrences.append((start, end, line_count))
+
+        # Plain string search
+        elif context_before or context_after:
+            # Context-aware search
+            pattern_str = ""
+            try:
+                if context_before and context_after:
+                    pattern_str = f"(?<={re.escape(context_before)})({re.escape(search)})(?={re.escape(context_after)})"
```

Pylint (3.3.7)

[convention] Line too long (120/100)

(C0301)